### ISSUE TITLE
Simple/Advanced mode

### ISSUE THREAD

**Issue by embryo10** [koreader/koreader#4116]
This is not some thing that I'm going to use for myself, just a suggestion for the KoReader community.

There were some posts at the MobileRead forums from Kobo users, saying more or less that the interface of KoReader is cumbersome, cluttered and confusing for a simple user.

So, I was wondering if it will be helpful to bring back the older Simple/Advanced mode for the menus.
Perhaps leaving the "lower" options as they are and minimizing the "upper" options to the bare essentials like:
 - from the first drop-down, keep only the TOC, Bookmarks and GoTo entries
 - from the second, only the Font and Hyphenation (maybe these belong to the lower menu as well)
 - from the third, the Frontlight, Network and Language
 - nothing from the fourth and fifth
 - and ditch the System statistics from the last one.

This is just a proposed configuration but I'm sure that you can come up with something much better..

**Comment by Frenzie** [comment]
Actually making things like this trivially simple was one of my MenuSorter goals, but I never ended up experimenting much  with different layouts myself. (But more as an enabler, not necessarily as doing it myself.)

**Comment by embryo10** [comment]
Yes, but after you make the customization, how are you going to switch between them?
Maybe with a menu preset somewhere?
